src/r1csfile.js

```diff
@@ -205,7 +205,7 @@ export async function readR1cs(fileName, loadConstraints, loadMap, singleThread,
     return res;
 }
 
-export async function readCustomGatesListSection(fd, sections) {
+export async function readCustomGatesListSection(fd, sections, fieldSize) {
```

Comment:
How do we decide if these go in r1csfile or binfileutils?

Reply:
Theses functions are specific for r1cs files